src/components/funFactPage/funFactPage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const FunFactPage: React.FunctionComponent = () => {
+    return (
+      <main>
+        <h1>FunFactPage</h1>
+        <p>FunFactPage</p>
+      </main>
+    );
+  };
```

Comment:
See how GitHub has added a little ⛔ at the bottom of the file here? This is to indicate that the file doesn't end with a single empty line, which is usually preferred. Could you add one here please?

Reply:
Done